conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -69,10 +69,8 @@ def __init__(self, conanfile, namespace=None, prefix="/"):
             # If cross-building and tools.android:ndk_path is defined, let's try to guess the Android
             # cross-building flags
             self.android_cross_flags = self._resolve_android_cross_compilation()
-            # Host triplet
-            if self.android_cross_flags:
-                self._host = self.android_cross_flags.pop("host")
```

Comment:
It was moved to line https://github.com/conan-io/conan/pull/17470/files#diff-22e59f4146297245137b23df3984e12f4cf4c6ad88d2186ab5543d65f39b18a0R123